### PR TITLE
Error messages

### PR DESCRIPTION
**Pull request by MorAbo**
added error message. 

#do notice that the work of erroring in generate exam is partial and doesnt cover all possible formik errors. i intend to deeply change the exam generating system along  with @RoiTief and therefore didnt want to put too much effort into irelevant code

### CODE REVIEW

File: ReactApp/src/components/errorMessage.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import React from 'react';
```
Comment (OfekN11) [inline ReactApp/src/components/errorMessage.js:1]: Why importing react?
Is this a componenet?

**Review by OfekN11 — COMMENTED** [review]
Hi did you read about error hadnling in next ?
https://nextjs.org/docs/app/building-your-application/routing/error-handling

I tried to use it, but with no success.
I ask it because by now you probably undertood the FE better than me, so it is worth asking

**Review by OfekN11 — COMMENTED** [review]
Wow it must has been frustrating...
Thanks!

File: ReactApp/src/constants.js [hunk 2]
```diff
@@ -95,17 +103,17 @@ export const CREATE_QUESTION = {
   SUBMIT_BUTTON: "Submit",
   VIEW_PDF_BUTTON: "View Question As PDF",
   STEM_REQUIRED: "Stem is required",
-  CORRECT_ANSWER_REQUIRED: "Correct answer text is required",
+  CORRECT_ANSWER_REQUIRED: "The key cannot be empty, please fill it or delete it",
   EXPLANATION_REQUIRED: "Explanation is required",
-  DISTRACTOR_REQUIRED: "Distractor text is required",
+  DISTRACTOR_REQUIRED: "Distractor text is required or delete this distractor",
```
Comment (RoiTief) [inline ReactApp/src/constants.js:108]: תשני אחד מהם כדי שיתאים לסטנדרט של השני. שניהם טובים אז לא אכפת לי מה תבחרי אבל שיהיה סטנדרט אחיד

**Review by idanah126 — APPROVED** [review]
Splitting to components in different files was very necessary and much easier to understand, nice work. Also using the same error handling mechanism throughout the pages was nicely done

**Pull request merged**